### ISSUE TITLE
The registration for method 'ConfigureFunctionsWebApplication' is expected for ASP.NET Core Integration.AZFW0014

### ISSUE THREAD

**Issue by jsquire** [Azure/azure-functions-dotnet-worker#2578]
# Issue Transfer

This issue has been transferred from the Azure SDK for .NET repository, [#44883](https://github.com/Azure/azure-sdk-for-net/issues/44883).

### Type of issue

Typo

### Description

[Enter feedback here]

### Page URL

https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.functions.worker.invocation?view=azure-dotnet

### Content source URL

https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/xml/ns-Microsoft.Azure.Functions.Worker.Invocation.xml

### Document Version Independent Id

691ab652-227b-0805-78fb-65933f19dd73

### Article author

@azure-sdk

### Metadata

* ID: 6d2612d8-4cb0-623b-e973-4d1b0b88d7b3

**Comment by kshyju** [comment]
@sivakarthick169 Could you please let us know which typo you noticed?

**Comment by varkey98** [comment]
As per the guide [here](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?tabs=windows#http-trigger), I've been trying to create a built-in model azure function, but always getting the error on the tiltle during build. Is this variant completely discontinued? I'm creating the templates from vs code and using dotnet 8 to build on mac.

**Comment by MisterRichards** [comment]
In my case, I had a nuget reference for 
**Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore**

I swapped that out for just a reference to:
**Microsoft.Azure.Functions.Worker.Extensions.Http**

and this error went away.  I was able to sort this out by comparing to THIS sample project:

https://github.com/Azure/azure-functions-dotnet-worker/tree/main/samples/FunctionApp

The upgrade paths on both Azure Function Apps and Azure SignalR is terrible (particularly for isolated model). You have my deepest sympathies.

**Comment by varkey98** [comment]
Yeah, I raised a separate issue, got the answer [there](https://github.com/Azure/azure-functions-dotnet-worker/issues/2594)

**Comment by mattchenderson** [comment]
Hi - this error message definitely needs improvement and should cite the inclusion of the package itself. We'll also update docs to make this easier to uncover.

Would you be able to share a bit more about your intent to remove the ASP.NET Core integration feature (or perhaps, why you specifically wanted `ConfigureFunctionsWorkerDefaults`)? The main motivation we see for this is when using .NET Framework, and the package should not have been added to the template in that case.

**Comment by varkey98** [comment]
I'm not writing the function itself, but a middleware for my vendor. And they can have functions of both types
